From 42c57c049054dfaa0be83f6721f7c4ce4a880e56 Mon Sep 17 00:00:00 2001
From: Ilya Sorochan <k0tran@altlinux.org>
Date: Fri, 6 Mar 2026 20:19:39 +0300
Subject: [PATCH 07803/15333] riscv: dts: starfive: jh7110-common: fix jh7110
 SoC boot from SD-card.

Add bootph-pre-ram to mmc1_pins clk-pins.

U-Boot pruned their overrides recently in favor of Linux which broke booting
from sd-card for me and Heinrich Schuchardt [1]. Pruning commit:
27f617019dd070cb61f2 ("riscv: dts: starfive: prune redundant jh7110-common overrides")

[1]  https://lore.kernel.org/all/ffdfc550-559b-4c59-9873-3f040fc3bb0e@canonical.com/

Signed-off-by: Ilya Sorochan <k0tran@altlinux.org>
Signed-off-by: Conor Dooley <mail@conchuod.ie>
---
 arch/riscv/boot/dts/starfive/jh7110-common.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/riscv/boot/dts/starfive/jh7110-common.dtsi b/arch/riscv/boot/dts/starfive/jh7110-common.dtsi
index a7a1c09a2c90..7d1eda67aeb5 100644
--- a/arch/riscv/boot/dts/starfive/jh7110-common.dtsi
+++ b/arch/riscv/boot/dts/starfive/jh7110-common.dtsi
@@ -438,6 +438,7 @@ GPOEN_ENABLE,
 			input-disable;
 			input-schmitt-disable;
 			slew-rate = <0>;
+			bootph-pre-ram;
 		};
 
 		mmc-pins {
-- 
2.55.0

